apps/mark-scan/backend/src/app.diagnostics.test.ts

```diff
@@ -225,6 +225,7 @@ test('saving the readiness report', async () => {
   const exportPath = exportResult.ok()![0];
   await expect(exportPath).toMatchPdfSnapshot({
     customSnapshotIdentifier: 'readiness-report',
+    failureThreshold: 0.001,
```

Comment:
I have some snapshot tests fail locally because of a 1 pixel difference on main, this fixes that